modules/swagger-codegen/src/main/resources/Javascript/es6/partial_model_generic.mustache

```diff
@@ -6,7 +6,7 @@
 * @module {{#invokerPackage}}{{invokerPackage}}/{{/invokerPackage}}{{#modelPackage}}{{modelPackage}}/{{/modelPackage}}{{classname}}
 * @version {{projectVersion}}
 */{{/emitJSDoc}}
-export default class {{classname}} {
+export default class {{classname}}{{#parent}}{{^parentModel}}{{#vendorExtensions.x-isArray}} extends Array {{/vendorExtensions.x-isArray}}{{/parentModel}}{{/parent}}{
```

Comment:
Let's move the space before `extends` or after `Array` out of the "if" clause, so there is space between `classname` and `{` when there is no `extends Array`.